### PR TITLE
External mysql db support

### PR DESCRIPTION
**Pull request by schmitmd**

### CODE REVIEW

File: scripts/populate.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-URL="http://localhost:9000/api/asset"
+URL="http://localhost:8080/api/asset"
```
Comment (byxorna) [inline scripts/populate.sh]: I thought the default port for `play run` is :9000?

**Comment by byxorna** [comment]
@schmitmd This all looks awesome, and has been a long time coming. Thanks! You probably need to sign the CLA if you haven't already: https://github.com/tumblr/collins/blob/master/CONTRIBUTING.md

File: scripts/populate.sh (hunk 1 above)
Comment (Primer42) [inline scripts/populate.sh]: Yeah, play def runs on port 9000 - @schmitmd can you revert this change?

**Comment by Primer42** [comment]
Also, it doesn't seem like we've received a CLA from you yet @schmitmd - sadly we can't merge these changes until you submit it.  Thanks.
https://github.com/tumblr/collins/blob/master/CONTRIBUTING.md

**Comment by schmitmd** [comment]
My mistake everyone.  Re: CLA, I've been wicked busy lately and hadn't gotten to it.  Finished it and submitted.  Created a new pull request without the 9000 port tweak.  I just use collins.sh, not play directly, so I thought it was just hanging around.
